frontends/ol-components/src/components/LearningResourceExpanded/InfoSectionV2.tsx

```diff
@@ -145,6 +155,77 @@ const InfoItemValue: React.FC<InfoItemValueProps> = ({
   )
 }
 
+const RunDates: React.FC<{ resource: LearningResource }> = ({ resource }) => {
+  const [showingMore, setShowingMore] = useState(false)
+  const asTaughtIn = showStartAnytime(resource)
+  const sortedDates =
+    resource.runs
+      ?.sort((a, b) => {
+        if (a?.start_date && b?.start_date) {
+          return Date.parse(a.start_date) - Date.parse(b.start_date)
+        }
+        return 0
+      })
+      .map((run) => formatRunDate(run, asTaughtIn)) ?? []
+  const showMore = allRunsAreIdentical(resource) && sortedDates.length > 2
```

Comment:
Just to check - it's possible that a single run date ("Start Date:" info item) can be displayed in addition to the differing runs table (several runs, same date, dates repeated in differing runs table and here)?

Reply:
I checked with @sovsey on this, and I had a misinterpretation here. In https://github.com/mitodl/hq/issues/5681, it says "Only courses that have multiple runs with DIFFERENT data display data in the table format" but doesn't say anything about hiding the start dates info item when that is the case. I've changed up the logic here so that the start dates info item is *not* shown if there are unique runs.